### DISCUSSION TITLE
V5 目前只可以在配置中的node的style里面设置自定义node吗？

### DISCUSSION THREAD

**Discussion by kuangdongksk** [antvis/G6#5961] [Q&A]
V5 目前只可以在配置中的node的style里面设置自定义node吗？这样不是只能使用一种自定义node吗，可不可以在node数据里边的style设置自定义节点
![image](https://github.com/antvis/G6/assets/75352559/c087a4b5-b416-467b-bf41-959304fb6b04)

**Comment by Aarebecca** [comment]
这是一个回调函数，你可以基于你的条件来返回不同的自定义节点呀

**Reply by kuangdongksk** [reply]
目前就是这么干的🙏
